apps/emqx/src/emqx_secret.erl

```diff
@@ -21,6 +21,10 @@
 %% API:
 -export([wrap/1, unwrap/1]).
 
+-export_type([t/1]).
```

Comment:
At least `debug_info` seems to include the changes.
```
5> beam_lib:chunks("_build/test/lib/emqx/ebin/emqx_secret.beam", [debug_info]).
{ok,{emqx_secret,[{debug_info,{debug_info_v1,erl_abstract_code,
                                             {[{attribute,{1,1},
                                                          file,
                                                          {"/emqx/apps/emqx/src/emqx_secret.erl",1}},
                                               {attribute,{19,2},module,emqx_secret},
                                               {attribute,{22,2},export,[{wrap,1},{unwrap,1}]},
                                               {attribute,{24,2},export_type,[{t,1}]},
                                               {attribute,{26,2},
                                                          type,
                                                          {t,{type,{26,15},union,[{var,{...},...},{type,...}]},
                                                             [{var,{26,9},'T'}]}}, ...
```

Yet my thought was that we don't have to hot reload this module, since runtime behavior is the same.

Reply:
appup files are generated automatically (see https://github.com/emqx/emqx/blob/master/scripts/update_appup.escript), so they can accidentally pick up this module. Maybe add a rule to this script to always ignore this module?

Reply:
> Maybe add a rule to the script to always ignore this module?

Good idea. Will do in a followup PR.

Reply:
Worth noting that apparently `debug_info` (`"Dbgi"`) section does not affect module MD5.
```
%% The following chunks are significant when calculating the MD5
%% for a module. They are listed in the order that they should be MD5:ed.

md5_chunks() ->
    ["Atom", "AtU8", "Code", "StrT", "ImpT", "ExpT", "FunT", "LitT", "Meta"].
```

And this is what `update_appup.escript` [uses](https://github.com/emqx/emqx/blob/0d6c4e0996588e550b18eaa0c032a6741ef6c27e/scripts/update_appup.escript#L600) to track module changes.